RFC/src/RFC-0201_TariScript.md

```diff
@@ -545,265 +660,269 @@ Once again, transaction fees are ignored to simplify the illustration.
 
 Alice owns \\( C_a \\) and provides the required script to spend the UTXO as was described in the previous cases.
 
-Alice needs a public key from Bob, \\( K\_{Sb} \\) to complete the one-sided transaction. This key can be obtained
+Alice needs a public key from Bob, \\( K_{Sb} \\) to complete the one-sided transaction. This key can be obtained
 out-of-band, and might typically be Bob's wallet public key on the Tari network.
 
-Bob requires the value \\( v_b \\) and blinding factor \\( k_b \\) to claim his payment, but he needs to be able to
+Bob requires the value \\( v_b \\) and blinding factor \\( k_b \\) to claim his payment, but he needs to be able to 
 claim it without asking Alice for them.
 
-This information can be obtained by using Diffie-Hellman and Bulletproof rewinding. If the blinding factor \\( k_b \\)
-was calculated with Diffie-Hellman using the sender offset keypair, (\\( k\_{Ob} \\),\\( K\_{Ob} \\)) as the sender
-keypair and the script keypair, \\( (k\_{Sb} \\),\\( K\_{Sb}) \\) as the receiver keypair, the blinding factor
-\\( k_b \\) can be securely calculated without communication.
+This information can be obtained by using Diffie-Hellman and Bulletproof rewinding. If the blinding factor \\( k\_b \\) 
+was calculated with Diffie-Hellman using the sender offset keypair, (\\( k\_{Ob} \\),\\( K\_{Ob} \\)) as the sender 
+keypair and the script keypair, \\( (k\_{Sb} \\),\\( K\_{Sb}) \\) as the receiver keypair, the blinding factor 
+\\( k\_b \\) can be securely calculated without communication.
 
-Alice uses Bob's public key to create a shared secret, \\( k_b \\) for the output commitment, \\( C_b \\), using
+Alice uses Bob's public key to create a shared secret, \\( k\_b \\) for the output commitment, \\( C\_b \\), using
 Diffie-Hellman key exchange.
 
 Alice calculates \\( k_b \\) as
-
 $$
 \begin{aligned}
-    k_b = k_{Ob} * K_{Sb}
-  \end{aligned}
- \tag{16}
+k_b = k_{Ob} * K_{Sb}
+\end{aligned}
+\tag{25}
 $$
 
-Next Alice next uses Bulletproof rewinding, see [RFC 180](RFC-0180_BulletproofRewinding.md), to encrypt the value
-\\( v*b \\) into the the Bulletproof for the commitment \\( C_b \\). For this she uses
-\\( k*{rewind} = \hash{k*{b}} \\) as the rewind_key and \\( k*{blinding} = \hash{\hash{k\_{b}}} \\) as the blinding
+Next Alice next uses Bulletproof rewinding, see [RFC 180](RFC-0180_BulletproofRewinding.md), to encrypt the value 
+\\( v_b \\) into the the Bulletproof for the commitment \\( C_b \\). For this she uses 
+\\( k_{rewind} =  \hash{k_{b}} \\) as the rewind_key and \\( k_{blinding} =  \hash{\hash{k_{b}}} \\) as the blinding 
```

Comment:
Why we cannot use `k_b` as the rewind/blinding key? (I forget)

Reply:
Considering [RFC-0180](https://rfc.tari.com/RFC-0180_BulletproofRewinding.html).

The security of the Bulletproof range proof in equations (2) and (3) are dependent on the condition that the five blinding factors used there should be sampled randomly from `Z_p`. If we were to only use `k_b`, it will result in using the same rewind nonce for `tilde(a)_j` and `tilde(s)_j` and the same blinding nonce for `tilde(t)_1j` and `tilde(t)_2j`, something that Dalek specifically commented on not to do.

Reply:
Ah thanks for the explanation :)